### ISSUE TITLE
[Chore] Ruby 2.7.0 changes

### ISSUE THREAD

**Issue by noraj** [picatz/shodanz#32]
```
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/shodanz-2.0.0/lib/shodanz/apis/streaming.rb:57: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/shodanz-2.0.0/lib/shodanz/apis/utils.rb:30: warning: The called method `slurp_stream' is defined here
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/shodanz-2.0.0/lib/shodanz/apis/utils.rb:36: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/shodanz-2.0.0/lib/shodanz/apis/utils.rb:168: warning: The called method `sync_slurp_stream' is defined here
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/shodanz-2.0.0/lib/shodanz/apis/utils.rb:170: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/shodanz-2.0.0/lib/shodanz/apis/utils.rb:113: warning: The called method `slurper' is defined here
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/async-http-0.38.3/lib/async/http/url_endpoint.rb:163: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/async-io-1.27.1/lib/async/io/host_endpoint.rb:98: warning: The called method `tcp' is defined here
```

Ruby 2.7

- Quick changelog: https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/
- Comprehensive changelog: https://rubyreferences.github.io/rubychanges/2.7.html
- Ruby 2.7 NEWS: Commentary by Cookpad’s Full Time Ruby Committers: https://sourcediving.com/ruby-2-7-news-commentary-by-cookpads-full-time-ruby-comitters-bdbaacb36d0c

The shodanz warning must be due to _Separation of positional and keyword arguments_

**Comment by picatz** [comment]
I'll look into this to come up with a fix 👍

**Comment by picatz** [comment]
Fixed in version [`2.0.1`](https://rubygems.org/gems/shodanz/versions/2.0.1).

**Comment by noraj** [comment]
According to https://github.com/socketry/async-http/issues/42, 0.50 fix the warning.

https://github.com/picatz/shodanz/blob/eaaa65dd969c3a5523053adc2d2c4fdc245f6af1/shodanz.gemspec#L23

I don't know if you want to update to `~>0.50`.

https://github.com/picatz/shodanz/blob/eaaa65dd969c3a5523053adc2d2c4fdc245f6af1/shodanz.gemspec#L24

For this one I tried with `1.27.1` without issue. So you may update to: `'>= 1.17.1', '< 1.27.1' `
Unfortunately this gem has no changelog.
